internal/pkg/cli/cmd/cmd.go

```diff
@@ -113,19 +141,32 @@ func NewRootCommand(stdin io.Reader, stdout io.Writer, stderr io.Writer, prompt
 
 	// Sub-commands
 	root.AddCommand(
-		local.StatusCommand(root),
-		sync.InitCommand(root),
-		local.ValidateCommand(root),
-		sync.PullCommand(root),
-		sync.PushCommand(root),
-		sync.DiffCommand(root),
-		local.PersistCommand(root),
-		local.FixPathsCommand(root),
-		local.EncryptCommand(root),
-		local.WorkflowsCommand(root),
-		local.CreateCommand(root),
+		sync.Commands(root),
+		local.Commands(root),
+		ci.Commands(root),
+		remote.Commands(root),
 	)
 
+	// Get all sub-commands by name
+	cmdsByName := make(map[string][]*cobra.Command)
+	visitSubCommands(root.Cmd, func(cmd *cobra.Command) {
+		if cmd.Parent() == root.Cmd {
+			// Skip top-level commands
+			return
+		}
+		cmdsByName[cmd.Name()] = append(cmdsByName[cmd.Name()], cmd)
+	})
+
+	// Create alias for all unique sub-commands.
+	// For example: "kbc init" can be used instead of "kbc sync init".
+	for _, cmds := range cmdsByName {
```

Comment:
`cobra` vyzaduje zadanie plnej cesty ku prikazu, napr. `kbc sync init`.
My chceme povolit aj skratky, ak je nazov prikazu unikazny. napr. `kbc init`.
To riesi tento kod.
Naklonuje dany prikaz ako `hidden` a vlozi ho priamo do `root` prikazu, tym vznikne skratka.